addons/test_discuss_full/tests/test_performance.py

```diff
@@ -960,12 +958,6 @@ def _get_init_messaging_result(self):
                     'operator_pid': (self.users[0].partner_id.id, 'Ernest Employee'),
                     'rtcSessions': [('insert', [])],
                     'seen_partners_info': [
-                        {
-                            'fetched_message_id': next(res['message_id'] for res in self.channel_livechat_2._channel_last_message_ids()),
-                            'id': self.channel_livechat_2.channel_member_ids.filtered(lambda m: m.partner_id == self.env.ref('base.public_partner')).id,
-                            'partner_id': self.env.ref('base.public_user').partner_id.id,
-                            'seen_message_id': next(res['message_id'] for res in self.channel_livechat_2._channel_last_message_ids()),
-                        },
```

Comment:
we are losing the fetch/seen info for the livechat channel?

Reply:
Yes, it is not available for guests